internal/privy/privy.go

```diff
@@ -0,0 +1,213 @@
+package privy
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/bluesky-social/indigo/api/agnostic"
+	"github.com/bluesky-social/indigo/api/atproto"
+	"github.com/bluesky-social/indigo/atproto/data"
+	"github.com/bluesky-social/indigo/lex/util"
+	"github.com/bluesky-social/indigo/xrpc"
+)
+
+// Privy is an ATProto PDS Wrapper which allos for storing & getting private data.
+// It does this by encrypting data, then storing it in blob. A special lexicon for this purpose,
+// identified by com.habitat.encryptedRecord, points to the blob storing the actual data.
```

Comment:
Should we get an actual domain to put our lexicons under? I've been using a personal domain I control just so "it's correct".